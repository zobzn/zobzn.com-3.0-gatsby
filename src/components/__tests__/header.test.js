import React from "react";
import renderer from "react-test-renderer";
import Header from "../header";

describe("Header", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Header />).toJSON();

    expect(tree).toMatchSnapshot();
  });

  it("renders correctly on homepage", () => {
    const tree = renderer
      .create(<Header location={{ pathname: "/" }} />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });

  it("renders correctly on inner page", () => {
    const tree = renderer
      .create(<Header location={{ pathname: "/about" }} />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
