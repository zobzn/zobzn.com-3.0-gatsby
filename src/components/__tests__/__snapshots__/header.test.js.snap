// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<header
  className="site-head"
>
  <a
    className="site-head__title"
    href="/"
  />
  <a
    className="zbz-link ml-3"
    href="/start"
  >
    Старт
  </a>
  <a
    className="zbz-link ml-3"
    href="/upwork"
  >
    Upwork
  </a>
</header>
`;

exports[`Header renders correctly on homepage 1`] = `
<header
  className="site-head"
>
  <h1
    className="site-head__title"
  />
  <a
    className="zbz-link ml-3"
    href="/start"
  >
    Старт
  </a>
  <a
    className="zbz-link ml-3"
    href="/upwork"
  >
    Upwork
  </a>
</header>
`;

exports[`Header renders correctly on inner page 1`] = `
<header
  className="site-head"
>
  <a
    className="site-head__title"
    href="/"
  />
  <a
    className="zbz-link ml-3"
    href="/start"
  >
    Старт
  </a>
  <a
    className="zbz-link ml-3"
    href="/upwork"
  >
    Upwork
  </a>
</header>
`;
